lib/datasource/github.js

```diff
@@ -47,16 +44,14 @@ async function getDependency(purl) {
   if (!versions) {
     return null;
   }
-  // Filter by semver if no versionScheme provided
-  if (options.clean === 'true') {
-    versions = versions.map(version => version.replace(/^v(\d)/, '$1'));
-  }
   const dependency = {
     repositoryUrl: 'https://github.com/' + repo,
-    versions: {},
   };
-  versions.forEach(version => {
-    dependency.versions[version] = {};
-  });
+  // TODO: should Github receive the versionScheme as input?
+  // versions should be already sorted by GIT
+  dependency.releases = versions.map(version => ({
+    version: version.replace(/^v(\d)/, '$1'),
```

Comment:
This datasource shouldn't filter `v` prefix *every time*. There may be package managers using the github datasource that depend on having the exact tag, and not a "cleaned" version of it.